posthog/urls.py

```diff
@@ -162,8 +162,8 @@ def authorize_and_redirect(request):
     path('api/user/redirect_to_site/', user.redirect_to_site),
     path('api/user/change_password/', user.change_password),
     path('api/user/test_slack_webhook/', user.test_slack_webhook),
+    path('decide/', decide.get_decide),
```

Comment:
How will the feature flags be stick for users?

Reply:
See comment above